frontend/src/employee-frontend/components/messages/MessageEditor.tsx

```diff
@@ -169,12 +176,27 @@ export default React.memo(function MessageEditor({
     },
     [i18n, draftState, draftWasModified]
   )
+  useEffect(
+    function updateReceiversOnSenderChange() {
+      const account = accounts.find(
+        (account) => account.id === message.sender.value
+      )
+      if (account?.type === 'PERSONAL') {
+        setReceiverTree(availableReceivers)
+      } else if (account?.type === 'GROUP') {
+        const groupId = account.daycareGroup.id
+        const selection = getSubTree(availableReceivers, groupId)
+        selection && setReceiverTree(selection)
```

Comment:
Just a style thing, but I prefer `if (selection) setReceiverTree(selection)` style when the code is about side effects and not about returning a value.